src/content/docs/usage/targets.mdx

```diff
@@ -0,0 +1,291 @@
+---
+title: Targets
+description: Learn how to manage Target configuration across Workspaces.
+sidebar:
+  label: Targets
+---
+
+import Label from '@components/Label.astro'
+
+<Label>
+  Distribution: **Open Source**
+</Label>
+
+A Target is an isolated processing unit where Workspaces are executed. It can operate on various infrastructures, including cloud-based and on-premises environments. Targets provide the necessary compute resources and meet security requirements within the defined infrastructure.
+
+## Create a Target
+
+Daytona provides an option to create a Target. Creating a Target requires creating a Target Configuration and specifying the Provider to use.
+
+1. Run the following command to create a Target:
+
+```shell
+daytona target create
+```
+
+:::note
+If you have not previously installed a Provider, registered a Runner, or created a Target Configuration, Dayona will prompt you to complete these steps before creating a Target.
```

Comment:
In line with the convo we had on Slack, let's assume that the user has Target Configs configured and they just need to choose it to create a target.

We leave creating the TC for the TC page